icu4c/source/i18n/unicode/udat.h

```diff
@@ -958,7 +958,25 @@ udat_getBooleanAttribute(const UDateFormat* fmt, UDateFormatBooleanAttribute att
 U_CAPI void U_EXPORT2
 udat_setBooleanAttribute(UDateFormat *fmt, UDateFormatBooleanAttribute attr, UBool newValue, UErrorCode* status);
 
+#ifndef U_HIDE_DRAFT_API
+/**
+ * Hour Cycle.
+ * @draft ICU 67
+ */
+typedef enum UDateFormatHourCycle {
+    /* hour in am/pm (0~11) */
```

Comment:
Please make these `/**` docs and use `@draft ICU 67` on each one, like you did in Java

Reply:
DONE